share/spice/get_events/get_events.js

```diff
@@ -18,7 +18,8 @@
                     sourceName: "GetEvents",
                     sourceUrl: 'http://getevents.co',
                     itemType: "Upcoming Local Events",
-                    formattedSourceName: 'More on GetEvents'
+                    formattedSourceName: 'More on GetEvents',
```

Comment:
While we're here should we change this "More **on**" to "More **at**"? Everything else on UI says "More at".